weaviate/gql/filter.py

```diff
@@ -613,6 +616,44 @@ def __str__(self):
         return f"where: {{operator: {self.operator} operands: [{operands}]}} "
 
 
+def _geo_range_to_str(value: dict) -> str:
+    """
+    Convert the valueGeoRange object to match `json` formatting.
+
+    Parameters
+    ----------
+    value : dict
+        The value to be converted.
+
+    Returns
+    -------
+    str
+        The string interpretation of the value in `json` format.
+    """
+    latitude = value["geoCoordinates"]["latitude"]
+    longitude = value["geoCoordinates"]["longitude"]
+    distance = value["distance"]["max"]
+    return f"{{ geoCoordinates: {{ latitude: {latitude} longitude: {longitude} }} distance: {{ max: {distance} }}}}"
+
+
+def _sanitize_str(value: str) -> str:
```

Comment:
personally I don't find too much docstring for little helper functions unhelpful but your decision

Reply:
I did this to maintain the style of the rest of this specific file, I certainly won't add this sorts of docstrings to the collections API code 😁

Reply:
alright, then let's merge this